extensions/mssql/src/errorDiagnostics/errorDiagnosticsConstants.ts

```diff
@@ -3,5 +3,20 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+import * as nls from 'vscode-nls';
+const localize = nls.loadMessageBundle();
+
 // Error code reference comes from here: https://learn.microsoft.com/en-us/sql/relational-databases/errors-events/database-engine-events-and-errors?view=sql-server-ver16
 export const MssqlPasswordResetErrorCode: number = 18488;
+export const MssqlCertValidationFailedErrorCode: number = -2146893019;
+
+export const ConnectionErrorDialogTitle = localize('connectionError', "Connection error");
+
+// Trust Server certificate custom dialog constants.
+export const TSC_ActionId = 'enableTrustServerCertificate';
+export const TSC_OptionName = 'trustServerCertificate';
+export const TSC_EnableTrustServerCert = localize('enableTrustServerCertificate', "Enable Trust server certificate");
+export const TSC_InstructionText = localize('trustServerCertInstructionText', `Encryption was enabled on this connection, review your SSL and certificate configuration for the target SQL Server, or enable 'Trust server certificate' in the connection dialog.
```

Comment:
@smartguest is this the right way to do multi-line resource string?

Reply:
This has been working - code is being moved over to extension..